src/gen/engine/src/main/java/com/paxful/sdk/PxsdkGenerator.java

```diff
@@ -0,0 +1,66 @@
+package com.paxful.sdk;
+
+import io.swagger.codegen.v3.*;
+import io.swagger.codegen.v3.generators.DefaultCodegenConfig;
+import io.swagger.codegen.v3.generators.typescript.TypeScriptFetchClientCodegen;
+import io.swagger.v3.oas.models.Operation;
+import io.swagger.v3.oas.models.media.Schema;
+import io.swagger.v3.oas.models.parameters.Parameter;
+
+import java.util.*;
+import java.io.File;
+
+public class PxsdkGenerator extends TypeScriptFetchClientCodegen {
```

Comment:
Let's add a short description of what this class does (e.g. adds doc block for parameters).

Reply:
done